OptionSelector/CQTableViewController.m

```diff
@@ -0,0 +1,105 @@
+//
+//  CQTableViewController.m
+//  OptionSelector
+//
+//  Created by Z on 8/13/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CQTableViewController.h"
+
+
+@interface CQTableViewController ()
+
+@end
+
+@implementation CQTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    CQModel *model = [[CQModel alloc] init];
+    [model setup];
+    
+    self.categories = model.categories;
+    
+    // Uncomment the following line to preserve selection between presentations.
```

Comment:
The first thing you should do when creating a new templated file like this is go through the commented out code included and either uncomment or delete it (and delete the explanatory comment). For readers of your code, it's just noise. (I know that I'm the only real reader of your code now, but it's good to establish good habits early).